### DISCUSSION TITLE
My Ultimate Basic Note Template

### DISCUSSION THREAD

**Discussion by AustinRedmon** [SilentVoid13/Templater#281] [Templates Showcase]
I press `ctrl + n` to create a note in a new pane (Templater automatically inserts this template since I toggled that on in the plugins options menu)
   - If I want to insert a different template instead I just hit `ctrl + a` and then insert a different template. I have `Templater: Insert Template` bound to `ctrl + e`

I also use Note Refactor and use a very similar template. I add `from: {link}` above `topics:` and just set `Note Link Template` to `{new_note_link}`. I have `Note Refactor: Extract selection to new note - first line as file name` bound to `ctrl + shift + n`

The `see also ` query is one that I created on my own. I believe that this query is very very powerful. It allows you find connections between notes inside the notes in an easily recognizable way. For the query to work you need to install the Dataview plugin. It returns linked mentions that are not included in the current file.

Let me know what you think.

````
---
tags: basic, new
aliases:
date created: <% tp.date.now() %>
---
%%
I add file links after topics, themes, and related

Why I use topics, themes, and related:
1. It gives me the flexibility of adding context and not having to include it directly in the note's content 
2. I create quite a bit of short, quick notes and so the ability to quickly add connections comes in handy a lot
3. I can come back to the note and add more connections very easily later on.
4. The added connections improves the graph view's ability to navigate, explore, and discover connections between your notes. 
%%
topics: 
themes: 
related: 

---
%%
templater plugin code to return the note's title when the template is inserted into the note
%%
### [[<% tp.file.title %>]]
- 

---
see also:
%%
returns list of linked mentions that are not present in the current note
%%
```dataview
list 
from [[#this.file.name]] and
!outgoing([[#this.file.name]])
```
````

**Comment by Training-Peter-Alexander** [comment]
Hi,

The dynamic commands like : <% tp.file.title %> does not work as expected. Korrect is:
<%+  tp.file.title %>
best regards 
Peter
